Assets/MixedRealityToolkit.SDK/StandardAssets/Shaders/FingerTipCursor.shader

```diff
@@ -0,0 +1,262 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+/// <summary>
+/// Note, this shader is generated from a tool and is not formated for user readability.
```

Comment:
Thank you for having the tool write out this note :)

One point of feedback on this is that for folks who are wanting to change this in the future, having some pointer/guidance on what that tool is (i.e. check out http://www.github.io/whatevertool) or otherwise some other docs/info about how it gets generated.

Otherwise the immediate action for someone who is doing this is to reverse engineer the process by doing random searches until the find the tool.

Reply:
Sadly the tool is a internal, custom one written by a member of the Shell team. I could reach out to them about open sourcing it, but that would probably take some time and effort. I could also spend a bit of time in the future re-formatting and commenting this shader for user readability (and for the hand triangle shader).

Reply:
Ah, yeah that would probably be more time than just now. 

I think that time is probably better spent here making the tool better (i.e. better formatting, maybe automatically adding comments where appropriate), than trying to fix up the output each time you regenerate it... it's a losing battle.